src/main.js

```diff
@@ -1,7 +1,7 @@
 /* istanbul ignore file */
 
 import Vue from "vue"
-import Tooltip from "vue-directive-tooltip"
+import VTooltip from "v-tooltip"
```

Comment:
remove vue-directive-tooltip then?

Reply:
yep i did!

Reply:
forgot the css though - will do that now.

Reply:
the reason i changed libraries was because the old library was not as supported and had some glitches that i couldn't overcome with the network selector. this new library is much better.

Reply:
damn, missed the removed line. all good. no issue with the cahnge.